policies/supporters_reporting.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Supporters Reporting
+created: 1 Jan 2025
+updated: 1 Jan 2025
+created_by: Jay Miller
+last_updated_by: Jay Miller
+---
+
+## A record of supporters to be kept
+
+A public record of supporters by year should be kept. This allows us to see an individual's contribution year over year.
+
+In order to protect the privacy of individuals, No information other than their name shall be made public.
+
+## Update/Removal of individuals
+
+Upon request in writing (via email to leadership@blackpythondevs.com), individuals can have one or more instances of their name updated or removed from the public record files. Considering the file is kept in version control, information cannot be completely deleted. To not have your name included please mark to be anonymous at time of donation.
+
+Record keeping is done using [CommitChange](https://commitchange.com). Information in CommitChange can be updated but should not be removed.
```

Comment:
```suggestion
You can request updates or removal of your name from our public records by sending a written request via email to **leadership@blackpythondevs.com**.

However, since we manage these records using version control, we cannot completely delete all information. So if you prefer not to have your name appear in our public records, please mark it as `anonymous` at the time of your donation.

We use [CommitChange](https://commitchange.com) for record-keeping. While we can update information in `CommitChange`, we cannot remove it entirely.
```